src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -1,758 +1,809 @@
-            }
-        },
-        onError: (err: Error) => {
-            // fail silently
-        }
-    }, undefined);
-
-    readonly copyNumberCountData = remoteData<CopyNumberCount[]>({
-        await: () => [
-            this.discreteCNAData
-        ],
-        invoke: async() => fetchCopyNumberData(this.discreteCNAData, this.molecularProfileIdDiscrete)
-    }, []);
-
-    @computed get sampleIds(): string[]
-    {
-        if (this.samples.result) {
-            return this.samples.result.map(sample => sample.sampleId);
-        }
-
-        return [];
-    }
-
-    readonly indexedHotspotData = remoteData<IHotspotIndex|undefined>({
-        await:()=>[
-            this.hotspotData
-        ],
-        invoke: ()=>Promise.resolve(indexHotspotsData(this.hotspotData))
-    });
-
-    @computed get mergedMutationData(): Mutation[][] {
-        return mergeMutations(this.mutationData);
-    }
-
-    @computed get mergedMutationDataIncludingUncalled(): Mutation[][] {
-        return mergeMutationsIncludingUncalled(this.mutationData, this.uncalledMutationData);
-    }
-
-    @computed get uniqueSampleKeyToTumorType(): {[sampleId: string]: string} {
-        return generateUniqueSampleKeyToTumorTypeMap(this.clinicalDataForSamples,
-            this.studiesForSamplesWithoutCancerTypeClinicalData,
-            this.samplesWithoutCancerTypeClinicalData);
-    }
-
-    @action("SetSampleId") setSampleId(newId: string) {
-        if (newId)
-            this._patientId = '';
-        this._sampleId = newId;
-    }
-
-    @action("SetPatientId") setPatientId(newId: string) {
-        if (newId)
-            this._sampleId = '';
-        this._patientId = newId;
-    }
-
-    @cached get mrnaExprRankCache() {
-        return new MrnaExprRankCache(this.mrnaRankMolecularProfileId.result);
-    }
-
-    @cached get variantCountCache() {
-        return new VariantCountCache(this.mutationMolecularProfileId.result);
-    }
-
-    @cached get discreteCNACache() {
-        return new DiscreteCNACache(this.studyToMolecularProfileDiscrete.result);
-    }
-
-    @cached get oncoKbEvidenceCache() {
-        return new OncoKbEvidenceCache();
-    }
-
-    @cached get genomeNexusCache() {
-        return new GenomeNexusCache();
-    }
-
-    @cached get pubMedCache() {
-        return new PubMedCache();
-    }
-
-    @cached get copyNumberCountCache() {
-        return new CopyNumberCountCache(this.molecularProfileIdDiscrete.result);
-    }
-
-    @cached get cancerTypeCache() {
-        return new CancerTypeCache();
-    }
-
-    @cached get mutationCountCache() {
-        return new MutationCountCache();
-    }
-
-    @cached get downloadDataFetcher() {
-        return new MutationTableDownloadDataFetcher(this.mutationData);
-    }
-
-    @action setActiveTabId(id: string) {
-        this.activeTabId = id;
-    }
-
-    @action clearErrors() {
-        this.ajaxErrors = [];
-    }
-
-}
+import * as _ from 'lodash';
```

Comment:
The comparison between old/new is messed up for this file. Could you point out what have been changed?

Reply:
@victoria34 could you give some info?